authors/apps/article/renderer.py

```diff
@@ -1,5 +1,6 @@
 import json
 from rest_framework.renderers import JSONRenderer
+from .models import LikeComment
```

Comment:
@mnswaleh You've imported the model but haven't used it in this file. Please remove it.
Also, revert the changes to spacing on this file.